### DISCUSSION TITLE
Recovery if something goes wrong

### DISCUSSION THREAD

**Discussion by tmontney** [terminatorul/NvStrapsReBar#116] [Q&A]
I haven't attempted this yet, but my motherboard supports updating the BIOS via a flash drive (which I will assume is USB Flashback). If for some reason this bricks the UEFI, will flashback save me?

**Comment by ashrambo** [comment]
Yes it is a trustable recover, but you need to create the correct USB image, it is better to have a programmer as CH341A, and it is cheap
